Description: wrong type in header
 <long description that can span multiple lines, optional>
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=749637
Forwarded: yes via ml
Last-Update: 2014-05-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/pdf.h
===================================================================
--- trunk.orig/pdf.h	2012-07-02 16:50:46.000000000 +0200
+++ trunk/pdf.h	2014-05-29 10:18:49.263943792 +0200
@@ -24,7 +24,7 @@
 #ifndef pdf_h
 #define pdf_h
 
-int print_pdf(FILE *s, const char *alreadyread, size_t len, const char *filename, int startpos);
+int print_pdf(FILE *s, const char *alreadyread, size_t len, const char *filename, size_t startpos);
 
 #endif
 
